eslint.config.js

```diff
@@ -89,6 +92,7 @@ export const config = ts.config(
     },
     rules: {
       'no-undef': OFF, // get rid of the 'React' is not defined error
+      'react-compiler/react-compiler': [WARN],
```

Comment:
I'm assuming this is because you want to see the impact? But is there a single rule in the thing?